components/sidebar/sidebar_channel/sidebar_channel.tsx

```diff
@@ -128,40 +140,63 @@ export default class SidebarChannel extends React.PureComponent<Props, State> {
             ChannelComponent = SidebarGroupChannel;
         }
 
-        return (
-            <Draggable
-                draggableId={channel.id}
-                index={channelIndex}
-            >
-                {(provided, snapshot) => {
-                    return (
-                        <li
-                            draggable='false'
-                            ref={this.setRef(provided.innerRef)}
-                            className={classNames('SidebarChannel', {
-                                collapsed: this.isCollapsed(this.props),
-                                unread: this.isUnread(),
-                                active: isCurrentChannel,
-                                dragging: snapshot.isDragging,
-                                fadeDMs: snapshot.isDropAnimating && snapshot.draggingOver?.includes('direct_messages'),
-                                noFloat: isDMCategory && !snapshot.isDragging,
-                            })}
-                            onTransitionEnd={this.removeAnimation}
-                            {...provided.draggableProps}
-                            {...provided.dragHandleProps}
-                            role='listitem'
-                            tabIndex={-1}
-                        >
-                            <ChannelComponent
-                                isCollapsed={this.isCollapsed(this.props)}
-                                channel={channel}
-                                currentTeamName={currentTeamName}
-                            />
-                        </li>
-                    );
-                }}
-            </Draggable>
+        const component = (
+            <ChannelComponent
+                isCollapsed={this.isCollapsed(this.props)}
+                channel={channel}
+                currentTeamName={currentTeamName}
+            />
         );
+
+        let wrappedComponent: React.ReactNode;
+
+        if (isDraggable) {
+            wrappedComponent = (
```

Comment:
It might be nice if we had some sort of `ConditionalDraggable` component that could toggle this behaviour via a prop, but I wasn't able to do that at this time.